bcitflex/scripts/scrape_and_load.py

```diff
@@ -91,8 +91,14 @@ def parse_offering_node(node: Node, course: Course, term: Term) -> Offering:
         instructor = instructor_node.text(False)
 
     # get price
-    price = node.css_first('li[class="sctn-block-list-item cost"]').text(False)
-    price = float(price.removeprefix("$") or 0)
+    price_node = node.css_first("li.sctn-block-list-item.cost div")
+
+    if price_node:
+        price_text: str = price_node.text(False)
+        price = float(price_text.strip().removeprefix("$") or 0)
+
+    else:
+        price = 0
```

Comment:
This brings up a question for me as to whether or not "no price" is a valid price for a course offering and if that means something different from `0`. What are you thoughts?

Reply:
@VleraBllacaa Sorry, I was a little quick with the merge, but I'd still be interested in your thoughts.

Reply:
@jonbiemond In my view, we can confidently assume that a course will always come with a specific cost. The previous issue arose when the site's structure was altered, causing it to consistently display a price of $0.

To enhance the user experience on our frontend, given that we anticipate having to pay a certain amount, excluding $0, I propose representing a $0 price with a hyphen ("-"). This change will make it less misleading for users, as they might otherwise think the course is free. Additionally, if you encounter a "-" in the price field on the table, it's a clear indicator that there might be script issues or alterations to the webpage's structure.

Reply:
@VleraBllacaa A quick check in the database confirms your assumption. I like your idea and have created an issue (#68), feel free to take it on!